Client/containers/Panel.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import CollectionModel from '../component/CollectionModel';
+var Menu = require('react-burger-menu').push;
+
+class Panel extends React.Component {
+
+
+
+  renderPanel() {
+    if (this.props.PanelMode === 'collection') {
+      return (
+        <div className='collection'>
+          {this.props.CollectionRestaurants.collection.map((restaurant) =>
+            <CollectionModel restaurant={restaurant} />
+          )}
+        </div>
+      );
+    } else if (this.props.PanelMode === 'filter') {
+      return (
```

Comment:
This should just be returning just the filter options, not the collection.  And it will need to map through all of the available filters, passing each of them the the specific filter they are tied to and the toggleFilter() method so they can be clicked.  Possibly include some other kind of  'if' to check if the filter is already selected so we can render it with an additional class.